### DISCUSSION TITLE
Welcome to newman-reporter-htmlextra Discussions!

### DISCUSSION THREAD

**Discussion by DannyDainton** [DannyDainton/newman-reporter-htmlextra#355] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with people who use and love using the reporter. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and new feature requests.
  * Engage with other like minded users.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by EuZak** [comment]
Hello!
I can't run reporter-htmlextra with docker.
I use wsl2, Ubuntu 20.04 in Win10.
I can get the report without using docker. 
If use docker:
$ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run /home/test/test.postman_collection.json -e /home/test/test_coll_env.json -r htmlextra
error: could not load environment
  ENOENT: no such file or directory, open '/home/test/test_coll_env.json'

**Comment by DannyDainton** [comment]
Can you raise this in its own discussion please and not in the comments of something different.
